test/spec/features/popup-menu/ReplaceMenuProviderSpec.js

```diff
@@ -188,6 +189,104 @@ describe('features/popup-menu - replace menu provider', function() {
       // then
       expect(domClasses(isCollectionMarker).has('active')).to.be.false;
     }));
+
+  });
+
+
+  describe('participants - multiplicity marker', function() {
+
+    beforeEach(bootstrapModeler(diagramXMLParticipants, { modules: testModules }));
+
+
+    it('should toggle on', inject(function(elementRegistry) {
+
+      // given
```

Comment:
The 
```
      // given
      var participant = elementRegistry.get('Participant_1');

      openPopup(participant);
```

is always the same for all 4 test cases isn't it? I suggest to therefore make it part of an "forEach" block. (also applies to the old tests above probably)

Reply:
I recall @nikku told me a few weeks back that when it came to testing, DRY wasn't the best guideline, as a test was supposed to be understandable as a standalone process, so it made sense to repeat some of the setup for each of them.

Reply:
Well when the goal is understandability, I would still argue that the understandability is not reduced when using a forEach block for this. The key content of the test is to click the icon, and that would still be included in the test hence would be understood.  
Anyway, this is just a suggestion which improves it IMO. But it is also okay the way it is.